net/extras/genInventoryFile.py

```diff
@@ -54,6 +54,9 @@ def writeInventory(self, outFd):
             outFd.write(var_line)
             var_line = "netplugin_if={} ".format(connInfo[node]['data'])
             outFd.write(var_line)
+            role = connInfo[node].get('role', 'worker')
```

Comment:
@neelimamukiri: Yes, they have higher precedence over role defaults. Role defaults are listed in roles/<role_name>/defaults/main.yml. What we have is role vars which would take a higher precedence.